plugin.json

```diff
@@ -2,14 +2,15 @@
     "id": "com.mattermost.plugin-splunk",
     "name": "Splunk",
     "description": "Splunk plugin for mattermost",
-    "version": "0.1.0",
+    "version": "1.0.0",
```

Comment:
Should we add `homepage_url` and `support_url`?

Reply:
Good catch, added.